resources/views/pages/documentation.blade.php

```diff
@@ -4,10 +4,11 @@
     'content',
     'banner'     => null,
     'isTutorial' => false,
+    'compact'    => false,
 ])
 
-<x-ark-container container-class="flex-col pt-10 pb-8 md:pt-12">
-    <div class="flex py-8 lg:divide-x divide-theme-secondary-200">
+<x-ark-container container-class="flex-col pb-8 {{ $compact ? 'pt-8 md:pt-8' : 'pt-10 md:pt-12' }}">
+    <div class="flex {{ $compact ? 'pb-8' : 'py-8' }} lg:divide-x divide-theme-secondary-200">
         <aside class="hidden flex-shrink-0 w-1/4 lg:block">
             <div class="overflow-y-auto sticky top-32 pr-10 h-sidebar custom-scroll">
```

Comment:
@crnkovic I think this `top-32` also has to go when there is no title above the sidemenu so it aligns with the content better